docs/README.md

```diff
@@ -1,15 +1,188 @@
-# Ansible VS Code Extension by Red Hat
-
-This extension adds language support for Ansible to
+# **Ansible VS Code Extension**
+
+The Ansible Extension for Visual Studio Code streamlines Ansible development by
+providing an integrated, feature-rich environment tailored for automation
+workflows. It offers features such as syntax highlighting, linting, intelligent
+code completion, and AI-assisted suggestions via Ansible Lightspeed. With
+support for multi-root workspaces, containerized execution environments, and
+extensive configuration options, the extension enhances productivity and ensures
+consistent code quality for both individual and team-based projects. This
+extension adds language support for Ansible in
 [Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=redhat.ansible)
-and [OpenVSX](https://open-vsx.org/extension/redhat/ansible) compatible editors
-by leveraging [ansible-language-server](als/README.md).
+and [OpenVSX](https://open-vsx.org/extension/redhat/ansible) by leveraging the
+[ansible-language-server](als/README.md).
+
+## **Installation Requirements**
+
+Before you begin, make sure your system has:
+
+- Ansible 2.9 or newer
+- Ansible Lint, unless linter support is disabled
+
+> **Note:** On Windows, use with the Remote - WSL or Remote - Containers
+> extensions for optimal compatibility.
+
+## **Getting Started**
+
+### Welcome Page
+
+The extension provides a comprehensive Welcome Page that serves as a dashboard
+for Ansible development tools. You can access it by:
+
+- Opening the Command Palette (Ctrl+Shift+P or Cmd+Shift+P)
+- Typing "Open Ansible Development Tools menu" and selecting it
+
+  ![Walkthrough](images/walkthrough.gif){ width=750 height=750 }
+
+  **OR**
+
+- Click on Getting Started
+
+  - Walkthroughs will appear on the right-hand side
+
+  ![Getting Started](images/getting_started.gif){ width=750 height=750 }
+
+The Welcome Page includes:
+
+- Quick access to create new playbooks, collections, and projects
+- System readiness status showing your Ansible environment configuration
+- Links to documentation and learning resources
+- Access to interactive walkthroughs
+
+### Interactive Walkthroughs
+
+The extension includes guided walkthroughs to help you get started with Ansible
+development:
+
+#### Create an Ansible environment
+
+- Create an Ansible playbook
+- Configure the tag in the status bar
+- Install the Ansible environment package
```

Comment:
Instead of having bullet points of the walkthroughs, can this be a 1-2 sentence summary of the walkthrough? Same for the other walkthroughs. This will help reduce docs overhead.